---
title: "Yue Wu - Research"
layout: textlay
excerpt: "Yue Wu -- Research"
sitemap: false
permalink: /Research/
---

# Research

1. **Discovering the connection between PreDM metabolic subtypes and real-time glucose data**. Continuous glucose monitoring (CGM) has enabled assessing real-time metabolic responses to activities (e.g., eating). The postprandial glycemic response (PPGR) to foods varies across individuals and is an independent risk factor for diabetes and cardiovascular diseases. However, the valuable information is often obscured by variances introduced by real-life activities, necessitating integration of computational methods and clinical domain knowledge. Using CGM data, we developed a computational model to classify PreDM subtypes defined by distinct physiological dysfunctions (e.g., insulin resistance and beta-cell dysfunction). Such a model has the potential to replace more expensive clinical tests and democratize personalized interventions (a). In addition, we integrated multi-omics data with CGM-measured PPGRs to standardized meals and identified participant subgroups with unique responses to foods, metabolic dysfunctions, and omics markers (b). For example, individuals with high PPGRs to potatoes exhibited higher insulin resistance and higher triglycerides. We further the analysis of CGM data through association with habit data (c) and package building (d). Our computational workflow provides a crucial initial step for patient subtyping of metabolic dysfunctions and precision diet intervention based on CGM.

    a. Metwally AA, Perelman D, Park H, **Wu Y**, Jha A, Sharp S, Celli A, Ayhan E, Abbasi F, Gloyn AL, McLaughlin T, Snyder M. Prediction of metabolic subphenotypes of type 2 diabetes via continuous glucose monitoring and machine learning. Nature Biomedical Engineering, 2024. PMCID: PMC12183321

    b. **Wu, Y.**, B. Ehlert, A.A. Metwally, D. Perelman, H. Park, A.W. Brooks, F. Abbasi, B. Michael, A. Celli, C. Bejikian, E. Ayhan, Y. Lu, S.M. Lancaster, D. Hornburg, L. Ramirez, D. Bogumil, S. Pollock, F. Wong, D. Bradley, G. Gutjahr, E.S. Rangan, T. Wang, L. McGuire, P. Venkat Rangan, H. Ræder, Z. Shipony, D. Lipson, T. McLaughlin, and M.P. Snyder, Individual variations in glycemic responses to carbohydrates and underlying metabolic physiology. Nature Medicine, 2025. 31(7): p. 2232-2243. PMCID: PMC12283382

    c. Park, H., A.A. Metwally, A. Delfarah, **Y. Wu**, D. Perelman, C. Mayer, C. McGinity, M. Rodgar, A. Celli, T. McLaughlin, E. Mignot, and M. Snyder, High-resolution lifestyle profiling and metabolic subphenotypes of type 2 diabetes. npj Digital Medicine, 2025. 8(1): p. 352. PMCID: PMC12159136

    d. Ehlert, B., D. Aron, D. Perelman, **Y. Wu**, and M.P. Snyder, Glucose360: An Open-Source Python Platform with Event-Based Integration for Continuous Glucose Monitoring Data Analysis. Diabetes Technology & Therapeutics, 2025. PMID: 40900178



2. **Automating time-series profiling of in vivo metabolism and discovering new regulatory processes**. Recent developments in omics approaches provide a comprehensive view of the biological system at one time point. However, the understanding of the time-series metabolic response to environmental perturbation is still limited in both experimental measurements and computational analysis. I co-led a nuclear magnetic resonance approach to collecting time-series metabolic data (c) and designed the computational method to efficiently extract chemical information from the high-dimensional dataset (b). This provided rich information regarding dynamic metabolic regulation under changing environmental conditions, and we applied the computational method to uncover processes related to in vivo carbon metabolism and glycogen utilization (a). We built a novel computational workflow to discover biochemical regulation from in vivo metabolomics data and applied it in multiple projects, including multi-omics in PreDM cohort studies and metabolism in cancer cells (d).

    a. **Wu Y**, Judge MT, Edison AS, Arnold J. Uncovering in vivo biochemical patterns from time-series metabolic dynamics. PLoS One. 2022;17:(5):e0268394. PMCID: PMC9098013

    b. **Wu Y**, Judge MT, Arnold J, Bhandarkar SM, Edison AS. RTExtract: time-series NMR spectra quantification based on 3D surface ridge tracking. Bioinformatics. 2020;36(20):5068-75. PMCID: PMC7755419

    c. Judge MT, **Wu Y**, Tayyari F, Hattori A, Glushka J, Ito T, Arnold J, Edison AS. Continuous in vivo Metabolism by NMR. Front Mol Biosci. 2019;6:26. PMCID: PMC6502900

    d. Cao S, Zhou Y, **Wu Y**, Song T, Alsaihati B, Xu Y. Transcription regulation by DNA methylation under stressful conditions in human cancer. Quantitative Biology. 2017 November; 5(4):328-337.



3. **Developing automatic phenotyping of biological systems through statistical modeling and deep learning**. Advancements in imaging and omics techniques enable the profiling of thousands of samples in a short time, which greatly promotes the phenotyping of plants, fungi, and human tissues. However, the necessary data annotation and information extraction are still manually intensive. We built multiple deep-learning frameworks to classify phenotypes (c and d) using residual neural networks and associate the image phenotypes with genomic information. We developed a program to automatically annotate and segment different symbiosis structures of Arbuscular mycorrhiza and stratify stages in the worm population (b). In addition to deep learning, We also modeled time series with Markov chain Monte Carlo and automatically extracted metabolic changes from living organisms (a). Our computational automation in phenotyping biological systems utilizes advanced statistical methods and significantly expands the sample size in association studies.

    a. **Wu Y**, Sanati O, Uchimiya M, Krishnamurthy K, Wedell J, Hoch J, Edison AS, Delaglio F. SAND: Automated Time-Domain Modeling of NMR Spectra Applied to Metabolite Quantification. Analytical chemistry. 2024;96:1843. PMCID: PMC10896553

    b. Zhang, S, **Wu Y**, Skaro M, Cheong JH, Bouffier-Landrum A, Torrres I, Guo Y, Stupp L, Lincoln B, Prestel A, Felt C, Spann S, Mandal A, Johnson N, Arnold J. Computer vision models enable mixed linear modeling to predict arbuscular mycorrhizal fungal colonization using fungal morphology. Sci Rep, 2024;14(1): p. 10866. PMCID: PMC11091061

    c. Krach EK, **Wu Y**, Skaro M, Mao L, Arnold J. Wild Isolates of Neurospora crassa Reveal Three Conidiophore Architectural Phenotypes. Microorganisms. 2020;8(11). PMCID: PMC7695285

    d. Krach EK, Skaro M, **Wu Y**, Arnold J. Characterizing the gene–environment interaction underlying natural morphological variation in Neurospora crassa conidiophores using high-throughput phenomics and transcriptomics. G3. 2022;12(4). PMCID: PMC8982394
